src/helpers.ts

```diff
@@ -995,3 +995,32 @@ export function vmSupportsExternalSnapshots(config: { capabilities?: HypervisorC
 export function vmHasVFIOHostDevs(vm: VM): boolean {
     return !!vm.hostDevices.find(hd => hd.driver === "vfio");
 }
+
+export interface QemuConf {
+    vnc_tls: boolean;
+    vnc_password: string | null;
+}
+
+const VNC_TLS_RX = /^[ \t]*vnc_tls[ \t]*=[ \t]*([0-9]*)/m;
+const VNC_PASSWORD_RX = /^[ \t]*vnc_password[ \t]*=[ \t]*"([^"\\]*(?:\\.[^"\\]*)*)"/m;
+
+export async function readQemuConf(): Promise<QemuConf> {
+    const conf: QemuConf = {
+        vnc_tls: false,
+        vnc_password: null,
+    };
+
+    const text = await cockpit.file("/etc/libvirt/qemu.conf", { superuser: "try" }).read();
+    if (!text)
+        return conf;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-5fe3bc4a-20250808-144213-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)